### DISCUSSION TITLE
Library for tasks or threads

### DISCUSSION THREAD

**Discussion by havok4u** [gleam-lang/gleam#3439] [General]
What is the best library for Tasks and/or threads.  I see rad, but it doesn't really seem to be actively maintained and if it were actively being used the error I get wouldn't manifest.

**Comment by inoas** [comment]
Did you check https://packages.gleam.run/?search=task
Did you ask on discord https://discord.gg/twY7ZhKTM3 ?

**Reply by havok4u** [reply]
I did not ask on discord as I don't have an account right now. This was listed as the discord alternate.  Midas looks interesting and seems to have active development, I'll look into that.  Thanks

**Reply by lpil** [reply]
Midas is a dependency/effect injection library, it is unrelated to threading/parallelism.

**Comment by inoas** [comment]
That said I'd like if there was something like https://getoban.pro/ for gleam without having to go through elixir.

**Reply by lpil** [reply]
Oban is a durable work queue library, it uses threading/parallelism but it's not a library for that.

**Comment by lpil** [comment]
Hi @havok4u !

Here on the Erlang VM we don't use threads, we use actors. The library you want is [gleam_otp](https://hexdocs.pm/gleam_otp/).

Rad is a command line task runner, it's unrelated to threading/parallelism.
